lms/djangoapps/courseware/models.py

```diff
@@ -248,22 +290,18 @@ class Meta:
     def __str__(self):
         return str(repr(self))
 
-    def save_history(sender, instance, **kwargs):  # pylint: disable=no-self-argument, unused-argument
+    def save_history(sender, instance, *args, **kwargs):  # pylint: disable=no-self-argument, unused-argument
```

Comment:
Oh, I see where you got the convention now. >_<

Okay, this was strangely defined, and I don't expect you to clean it up. But please rename the superclass's `save_history` to something else so that there's no weird inheritance shadowing of methods with different signatures.

Reply:
Renaming the parent staticmethod to `save_history_entry`